web/models/stable_diffusion/main.py

```diff
@@ -92,11 +92,28 @@ def save_output_img(output_img):
         )
     else:
         out_img_path = Path(generated_imgs_path, f"{out_img_name}.png")
-        output_img.save(out_img_path, "PNG")
+        pngInfo = PngImagePlugin.PngInfo()
+
+        if args.write_metadata_to_png:
+            # set height and width.
+            height = 512  # default height of Stable Diffusion
+            width = 512  # default width of Stable Diffusion
```

Comment:
I guess eventually we will need to set these based on the chosen width/height.

Reply:
Indeed, it felt a little awkward copy pasting hard coded values. 😅